index.html

```diff
@@ -30,76 +30,250 @@ <h2 class="logo">
+          <div class="works-tonic">
+            <div class="snapshoot">
+              <img src="./images/Snapshoot-1.png" alt="tonich4" />
+            </div>
+            <div class="work-title">
+              <h4>Tonic</h4>
+              <ul class="works-info">
+                <li class="works-ref-font">CANOPY</li>
+                <li><span class="counter"><img src="./images/counter.png" alt="counter" /></span></li>
+                <li>Back End Dev</li>
+                <li><span class="counter"><img src="./images/counter.png" alt="counter" /></span></li>
+                <li>2015</li>
+              </ul>
+              <p class="text-tonic">
+                A daily selection of privately<br>
+                personalized reads; no accounts or<br>
+                sign-ups required.</p>
+              <ul class="skills">
+                <li class="li-html">html</li>
+                <li class="li-css">css</li>
+                <li class="li-javascript">javascript</li>
+              </ul>
+            </div>
+
+            <div class="link-project"></div>
+            <a class="linktonic" href="#">See Project</a>
+          </div>
         </div>
-        <p class="tonic">
-          A daily selection of privately<br>
-          personalized reads; no accounts or<br>
-          sign-ups required.</p>
-
-        <ul class="skills">
-          <li>html</li>
-          <li>css</li>
-          <li>javascript</li>
-        </ul>
-        <button type="button" class="linktonic">See Project</button>
       </article>
 
     </section>
 
+    <section class="about">
+      <div class="container-about">
+        <div class="back-about">
 
-    <footer>
+          <h1>About<br>Myself</h1>
+          <p class="intro-text">I’m a software developer! I can help you
+            build a product, feature or website. Look
+            through some of my work and
+            experience! If you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <a href="#">
+            <h3 class="connectlink">LET´S CONNECT</h3>
+          </a>
+
+          <div class="socialmedia">
+            <ul class="icons">
+              <li>
 
+                <a href="#"><img src="./images/twitter.svg" alt="twitter"></a>
+              </li>
+              <li>
+
+                <a href="#"><img src="./images/linkedin.svg" alt="linkedin"></a>
+              </li>
+              <li>
+
+                <a href="#"><img src="./images/m.svg" alt="M"></a>
+              </li>
+              <li>
+
+                <a href="#"><img src="./images/github.svg" alt="Git"></a>
+              </li>
+              <li>
+
+                <a href="#"><img src="./images/vector.svg" alt="V"></a>
+              </li>
+            </ul>
+          </div>
+          <div class="link-project"></div>
+          <a class="linktonic" href="#">Get my resume</a>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] I can't see  `get my resume ` button please try to fix that in order to move to the next milestone 
![image](https://user-images.githubusercontent.com/61790587/171292387-fcd572b4-92cf-4949-bb63-f89557f074c8.png)